### DISCUSSION TITLE
Git Tags and/or Releases

### DISCUSSION THREAD

**Discussion by mytlogos** [TechnitiumSoftware/DnsServer#739] [General]
I do not seem to have a good way to track new releases of this wonderful project.
There exists the [technitium archive](https://download.technitium.com/dns/archive/) where i could look for new releases periodically, the [blog](https://blog.technitium.com/), where major announcements are made and [docker hub](https://hub.docker.com/r/technitium/dns-server), where each released image is tagged.

But i do not have a way to "listen" or be notified on a new release.

One way this can be solved is by creating git tags and/or releases and pushing them to github.
Github allows creating a watcher for Releases on repositories, which also may send mails if one wants.
Github also provides a RSS feed for pushed git tags and/or created releases, so creating a release would fit quite nicely in my current setup.

Providing git tags is also nice, as one could be sure which commits are currently in a release (and which are not).

**Comment by ShreyasZare** [comment]
Thanks for the suggestion. Will check out git tags.

**Comment by ShreyasZare** [comment]
I have added tag for latest release. Will add tags for subsequent releases.
